ingest/ingest-people.py

```diff
@@ -338,7 +359,8 @@ def get_courses(coursesgraph):
             .one().value
         if course:
             print("course exists: ", course)
-            courses.append({"course": course})
+#            courses.append({"course": course})
```

Comment:
Is there a reason to comment out this code vs just removing it? Can't we 'backtrack' the change if needed using git?

Reply:
agreed, will remove